src/app/pages/enter-reset-password/enter-reset-password.component.html

```diff
@@ -3,8 +3,8 @@
     <div class="col-md-3"></div>
     <div class="col-md-6">
 
-      <img class="img-fluid align-middle center" src="assets/img/ICON/Logo.png" alt="landville-logo" width="150"
-        height="95" />
+      <img class="img-fluid align-middle center" src="../../../assets/img/ICON/Logo.png" alt="landville-logo"
```

Comment:
Please make this absolute

Reply:
Thanks, @patrickf949, I have worked on this